framework/atoms/fields/fields.html

```diff
@@ -0,0 +1,37 @@
+<div class="field">
+  <div class="field__label">Name</div>
+  <div class="field__items">
+    John Doe
+  </div>
+</div>
+<div class="field">
+  <div class="field__label">Address</div>
+  <div class="field__items">
+    Street name, Number 54, Brussels
+  </div>
+</div>
+<div class="field">
+  <div class="field__label">Phone number</div>
+  <div class="field__items">
+    +48 531 555 003
+  </div>
+</div>
+<div class="field">
+  <div class="field__label">Date</div>
+  <div class="field__items">
+    <div class="field__item field__item--expired">
+      12 October 2016 (CET)
+    </div>
+  </div>
+</div>
+<div class="field social-media-links--field field-label-inline clearfix">
+  <div class="field__label">Social media</div>
+  <ul class="field__items social-media-links__list social-media-links__horizontal">
+    <li class="social-media-links__item icon-social--smaller">
```

Comment:
Yup, good catch. I'll implement "social media links" and then I'll come back to this one